src/components/Datatable/ControlModule/ControlModule.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { isNotEmpty } from 'ramda-adjunct';
+import styled from 'styled-components';
+
+import { getColor } from '../../../utils/helpers';
+import { Col } from '../../layout';
+import { FlexContainer } from '../../FlexContainer';
+import { BatchModule } from '../BatchModule';
+import { SearchBar } from '../../forms/SearchBar';
+import { ToolsTabItem } from '../ToolsTabItem';
+import { ControlModuleProps } from './ControlModule.types';
+import { Filters } from '../../Filters';
+import { renderSuggestionFilter } from '../../forms/SearchBar/SearchSuggestionFormats';
+
+const ControlModuleContainer = styled(FlexContainer)`
+  background-color: ${getColor('graphite3H')};
+`;
+
```

Comment:
Suggestion:
1) let's create ToolBar component and encapsulate the toolTabsItems in instance
2) remove this toolTabsItems completely
3) define toolTabsItems in map in the ToolBar component (do not pass anything from outside exept some custom handlers for activate/deactivate), like:
```js
const tools = [
  {
    label: '',
    icon: ''
  },
  ...
]
```
4) move `useEffect` into the ToolBar component
5) for show/hide filters panel we should use DatatableContext and should be controled in the ToolBar component

Reply:
6. for the Toolbar I would add `state` property which defines initial state, e.g. filters are opened, initial show/hide state...

Reply:
I am confused by 3. Why do you think the tools icons/labels is better to be hardcoded and not flexible for future tables with different tools?